framework/vkdf-init.cpp

```diff
@@ -575,7 +489,7 @@ _init_swap_chain(VkdfContext *ctx)
    swap_chain_info.preTransform = present_transform;
    swap_chain_info.compositeAlpha = VK_COMPOSITE_ALPHA_OPAQUE_BIT_KHR;
    swap_chain_info.imageArrayLayers = 1;
-   swap_chain_info.presentMode = present_mode;
+   swap_chain_info.presentMode = VK_PRESENT_MODE_FIFO_KHR;
```

Comment:
NAK -- Mesa is not the only Vulkan implementation available even if it is the one we use :)